services/mentor_service.js

```diff
@@ -0,0 +1,48 @@
+const repo = require('../repositories/mentor_repository');
+const mailerService = require('../services/mailer_service');
+const logger = require('../logger')(__filename);
+
+async function getMentors(userId, status){
+  return await repo.getMentors(userId, status);
+}
+
+async function getMentorById(userId, mentorId){
+    
+  return await repo.getMentorById(userId, mentorId);
```

Comment:
I think we also agreed on some filtering conditions. I.e.:

Getting active/inactive mentors can be performed by logged in and not logged in users, the data will vary:
- Name, description, tags (photo - in future) - publicly
- Other stuff (contact info) - for logged in users only